src/routes/indexable-indexation-route.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Reindexation route for indexables
+ *
+ * @package Yoast\WP\SEO\Routes\Routes
+ */
+
+namespace Yoast\WP\SEO\Routes;
+
+use WP_REST_Response;
+use Yoast\WP\SEO\Actions\Indexation\Indexable_Post_Indexation_Action;
+use Yoast\WP\SEO\Conditionals\No_Conditionals;
+use Yoast\WP\SEO\Main;
+
+/**
+ * Indexable_Reindexing_Route class
```

Comment:
Okay, I feel like you get the point, and again I wonder why PHPCS is not picking this up...
```suggestion
 * Indexable_Reindexing_Route class.
```